src/docs/fundamentals/ElectronicLabNotebooks.md

```diff
@@ -1,53 +1,140 @@
+Every lab builds on a collection of protocols. Some protocols may describe routinely performed _standard operating procedures_, while others may be continuously evolving and adapted to a specific use-case. In pre-digital times, protocols for a scientific project were collected in classical pen-and-paper lab notebooks. By design, the metadata knowledge collected in these single-copy protocols was hard to access (hidden in a shelf), maintain (over generations of researchers) or share and communicate (hand-written notes, post-its) with peers during the ongoing scientific process. <!-- (unless published) -->
 
-## ELN - Digital Protocol Editors
-
-Once a protocol is digitalized, the information that used be stuck in a single-copy paper can be replicated with ease, facilitating quicker collaboration, sharing, and reuse (of those laboratory-central processes). This can be done via a combination of text software, local or cloud servers, wiki platforms or by using a specialized software - the **electronic** lab notebook (ELN).
+Also, a lot of data in research is gathered in a digital form, which is not very suitable for an analog notebook.
+Furthermore, there is no backup for the notebook or any other option to access the protocols if the notebook goes missing.
 
-ELNs were developed (designed) to help overcome the limitations of classical pen-and-paper lab notebooks and align well with daily wet-lab routines. ELNs facilitate the editing and sharing of protocols, usually in a closed environment such as a lab or consortium.
+## ELN - Digital Protocol Editors
 
-Designed as lightweight (low-burden, free-text) protocol editors, ELNs are a major step towards digitalization of laboratory processes.
+All of these Problems can be solved by an Electronic Lab Notebook (ELN).
+You can write down all necessary Information in combination with all kind of files like images, tables, documents and so on.
+You can easily link between all kind of data and notebook entries.
+This creates [FAIR][knowledge-fair] protocols for your research and is a major step towards the digitalization of laboratory processes.
+
+- **F**indable, every entry and piece of data is findable through a search function and linkage
+- **A**ccessible, access the notebook from everywhere, manage access rights and share data for good collaboration
+- **I**nteroperable, an ELN uses common formats usable on any computer, making it easy interchangeable; you can also create interfaces between your ELN and your instruments or repository's to have a better workflow in your research
```

Comment:
Common formats (basically pure text), but not really machine-interpretable.

Reply:
many elns have some kind of APIs. With it you can make the content machine-interpretable. You can also export the data most of the time to json